src/main/groovy/org/checkerframework/gradle/plugin/CreateManifestTask.groovy

```diff
@@ -0,0 +1,44 @@
+package org.checkerframework.gradle.plugin
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+
+/**
+ * This task generates a manifest file in the style of
+ * https://checkerframework.org/manual/#checker-auto-discovery
+ * Once the plugin places this directory containing it on the annotation processor path,
+ * javac can find the checkers via annotation processor discovery.
+ */
+class CreateManifestTask extends DefaultTask {
+
+    private final static String buildDirectory = "checkerframework/META-INF/services/"
```

Comment:
Could you add ${project.buildDir} here instead of in two places below?  Also, shouldn't this be manifestDirectory rather than buildDirectory?

Reply:
No. These are static constants, but `${project.buildDir}` is a property of the project that will be different between instantiations of this class.